### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
## Learning objectives
- Run a program using the command line. ✔
- Use Ruby syntax for basic programming operations. ✔
- Use Ruby to solve basic coding challenges. ✔

## Project requirements

- Create a method to decode a Morse code character. ✔
- Create a method to decode an entire word in Morse code. ✔
- Create a method to decode the entire message in Morse code. ✔

### CODE REVIEW

File: decode.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  full_word = ''
+  split_word = word.split
+  split_word.each_with_index do |t, _i|
+    @morse.each_with_index do |m, idx|
+      next unless t == m
+
+      @alphabet.each_with_index do |letter, i|
+        full_word += letter if i == idx
+      end
+    end
```
Comment (helman101) [inline decode.rb]: - [x] Your `decode_word` function should reuse the `decode_char` in order to meet the DRY(Don't repeat yourself) requirement, kindly fix this 😄

File: decode.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@morse = '.-', '-...',	'-.-.',	'-..',	'.',	'..-.',	'--.',	'....',	'..',	'.---',	'-.-',
```
Comment (helman101) [inline decode.rb]: - [x] Please create a third function called `decode`, as the project requirement said:
    - `Create a method to decode the entire message in Morse code, take a string parameter, and return the string representation. Every word will be separated by 3 spaces`
    - This method should use `decode_word` function. 🚀

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @xJokeer & @Laylo309 👋, 

Nice job so far with this project 😄, but there are still some changes to do.

## Required Changes ♻️

- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**